lib/Zef/Repository/Ecosystems.rakumod

```diff
@@ -98,7 +102,7 @@ class Zef::Repository::Ecosystems does PackageRepository {
         my @candidates = @!distributions.map: -> $dist {
             Candidate.new(
                 dist => $dist,
-                uri  => ($dist.source-url || $dist.meta<support><source>),
+                uri  => ($dist.source-url || try $dist.hash<support><source>),
```

Comment:
https://github.com/ugexe/zef/blob/1cbd322f6da5e1b67245dae98751463919a0229c/lib/Zef/Repository/Ecosystems.rakumod#L159
this is what it should be